setup.py

```diff
@@ -26,7 +26,7 @@
             "tldr = tldr:cli"
         ]
     },
-    install_requires=['termcolor', 'colorama', 'argcomplete'],
+    install_requires=['termcolor', 'colorama', 'shtab>=1.3.6'],
```

Comment:
```suggestion
    install_requires=['termcolor', 'colorama', 'shtab>=1.3.10'],
```

optional for full bash support